query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/QueryRequestToPinotSQLConverter.java

```diff
@@ -125,107 +137,99 @@
     return new SimpleEntry<>(pqlBuilder.toString(), paramsBuilder.build());
   }
 
-  private String convertFilter2String(
+  private boolean isGroupingSelectionForMapAttribute(
```

Comment:
With the new changes as Ronak pointed out over [here](https://github.com/hypertrace/query-service/pull/107#discussion_r760063277), this could be removed. Forgot to do that.

Reply:
resolved